db/schema.rb

```diff
@@ -49,6 +49,7 @@
     t.datetime "updated_at"
     t.datetime "applied_at"
     t.integer  "updater_id"
+    t.text     "state",                       default: "draft", null: false
```

Comment:
Sorry, saw your comment too late. I'll revert 8f6eefef.

Actually, I was able to get rid of most of the `state` calls, instead calling e.g. `applied?` directly on a draft. The only place where `state` is still used is when it's displayed in a template (`application_drafts_helper.rb`, line 12)